Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/workspace.xml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 480e7e0bac861037eb37ea24d5455ee51a44a35d)
+++ b/.idea/.gitignore	(date 1650535885969)
@@ -1,1 +1,2 @@
-/workspace.xml
\ No newline at end of file
+/workspace.xml
+shelf/
\ No newline at end of file
